CONTRIBUTING.md

```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
 * Note: if you have write access to the repository, do not directly merge pull 
   requests. Let another team member review your pull request and approve it.
 
+## Before Merge
+* Bump the version into the files; `npm-shrinkwrap.json`, `package.json`, `sonar-project.properties`, e.g. version": "3.2.0-BUILD-NUMBER"
+* Prepend your commit message with a JIRA ticket number and the text Bump Version, e.g. "FH-1234: Bump Version."
+* Ensure that the commits into the PR are properly squashed and you have at list one commit for the changes and one for the bump version.
```

Comment:
into -> in

Reply:
so, you need a minimum of two commits? if so, this text if fine

Reply:
@finp yep 👍 tks 
@pb82 Done:-)